frontend/src/components/Comment/Comment.js

```diff
@@ -12,7 +13,11 @@ export const Comment = (props) => {
     return (
         <div className="comment-div">
             <br />
-            <Link to={`/account/${props.comment.account}`}>{props.comment.account} </Link> {props.comment.comment}
+            <Link to={`/account/${props.comment.account}`}>{props.comment.account} </Link> 
+            {Boolean(props.comment.editable) ? <EditComment {...props} commentId={props.comment.id} ></EditComment>: ''}
```

Comment:
For the sake of consistency, please add a tab before the line starting with  `{Boolean(props.comment.editable)...}`